mappings/net/minecraft/client/tutorial/TutorialManager.mapping

```diff
@@ -3,7 +3,8 @@ CLASS net/minecraft/class_1156 net/minecraft/client/tutorial/TutorialManager
 	FIELD field_5646 currentHandler Lnet/minecraft/class_1155;
 	METHOD <init> (Lnet/minecraft/class_310;)V
 		ARG 1 client
-	METHOD method_4905 getGameMode ()Z
+	METHOD method_4905 isClientSurvival ()Z
```

Comment:
Since this is in a client class, maybe `is(In)Survival` would be enough?

Reply:
Sure